addons/mail/models/ir_attachment.py

```diff
@@ -42,6 +42,10 @@ def _delete_and_notify(self):
                 self.env['bus.bus']._sendone(self.env['mail.channel'].browse(attachment.res_id), 'ir.attachment/delete', {
                     'id': attachment.id,
                 })
+            elif attachment.res_id:
+                self.env['bus.bus']._sendone(self.env.user.partner_id, 'ir.attachment/delete', {
+                    'id': attachment.id,
+                })
```

Comment:
only the first param of `_sendone` (the target) should be conditional, not the rest of the message
also condition on `res_id` is not necessary on the generic case, the attachment id is enough (`res_id` is channel id)

Reply:
It should be good now